atat/routes/admin/admin.py

```diff
@@ -0,0 +1,61 @@
+from flask import Blueprint, render_template, request
+
+from atat.domain.authz.decorator import user_can_access_decorator as user_can
+from atat.domain.portfolios import Portfolios
+from atat.domain.users import Users
+from atat.forms.ccpo_user import CCPOUserForm
+from atat.forms.data import SERVICE_BRANCHES
+from atat.models import Permissions, User
+from atat.utils.context_processors import atat as atat_context_processor
+
+bp = Blueprint("admin", __name__)
+bp.context_processor(atat_context_processor)
+
+
+def get_portfolios_from_user(user: User = None):
+    if user is None:
+        return ""
+
+    portfolios = Portfolios.for_user(user)
+    return [(portfolio.displayname, portfolio.portfolio_id) for portfolio in portfolios]
+
+
+def service_branch_label(service_branch: str = None):
+    if service_branch is None:
+        return ""
+
+    labels = list(filter(lambda x: service_branch in x, SERVICE_BRANCHES))
+    if len(labels) > 0:
+        return labels[0][1]
+
+    return ""
```

Comment:
This might read a little more simply with a list comprehension and a comment.
```suggestion
def service_branch_label(service_branch: str = None) -> str:
    """
    Get each user's service branch.
    
    If the service branch is set properly, the name of that branch is returned,
    otherwise, an empty string will be returned.
    """
    if not service_branch:
        return ""

    # The first item in each tuple is the key, which is passed in. Each
    # second item is the actual name of the service branch.
    matching_branch_names = [
        known_branch[1]
        for known_branch in SERVICE_BRANCHES
        if service_branch == known_branch[0]
    ]

    if not matching_branch_names:
        return ""
    return matching_branch_names[0]
```

Please add docstrings (as in the above) to any new methods.